### PR TITLE
Upgrade Grpc.Core packages to 2.34.0-pre1

### PR DESCRIPTION
**Pull request by jtattermusch**
It has some fixes such as https://github.com/grpc/grpc/pull/24768

### CODE REVIEW

**Comment by jtattermusch** [comment]
I'll cut the 2.34.x release branch after this is merged.

**Review by JunTaoLuo — APPROVED** [review]

**Comment by JamesNK** [comment]
Ready to merge and branch after https://github.com/grpc/grpc-dotnet/pull/1121 is merged.

**Review by JamesNK — APPROVED** [review]

**Pull request merged**
